src/components/SideBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+
+function SideBar() {
+  return (
+    <ListGroup defaultActiveKey="/">
+      <ListGroup.Item action href="/" variant="warning">
+        LISTINGS
```

Comment:
- [ ] Good job on implementing the side bar 👏🏻 it looks great, but based on project requirements we should uphold the provided design. Therefore, I suggest giving side navbar full height, add logo on the top and add social links on the bottom. Delete the top bar with logo. 

Your design  | Provided design
------------- | -------------
<img width="369" alt="Screenshot 2022-04-06 at 17 15 55" src="https://user-images.githubusercontent.com/77038610/161983266-eb29649b-557c-4513-b013-bfd54dbb9200.png"> | <img width="168" alt="Screenshot 2022-04-06 at 17 17 05" src="https://user-images.githubusercontent.com/77038610/161983540-281b7a9b-eb5c-49ea-b63b-e503de4571f8.png">

- [ ] Lets also be consistent with colors. I am using these colors for reservations page: 

```
 white-background: #fff;
 green-background: #97bf02;
  border-left: 2px solid #c4dd78;
```